secant/Generated/XCAssets+Generated.swift

```diff
@@ -30,6 +30,10 @@ internal enum Asset {
     }
   }
   internal enum Colors {
+    internal enum Background {
+      internal static let linearGradientEnd = ColorAsset(name: "LinearGradientEnd")
```

Comment:
Given the new naming convention, this would be screenBackgroundGradientEnd and screenBackgroundGradientStart, and the Asset names would be updated as well.